### DISCUSSION TITLE
GLOSSARY - Bargain Purchase Value

### DISCUSSION THREAD

**Discussion by JachymHercher** [eprocurementontology/eprocurementontology#493] [Ideas]
Term: Bargain Purchase Value
Definition: The financial worth of a bargain purchase, taking advantage of a particularly advantageous opportunity available for a very short time at a price considerably lower than market prices.

This concept, understandably, overlaps with the concept of "Value". To increase clarity and reduce duplication, would't it be better to separate them? In my opinion, the concept "Value" can exist as usual. Then, this concept could be renamed to "Bargain Purchase" with a definition of something like "This was a bargain purchase, taking advantage of a particularly advantageous opportunity available for a very short time at a price considerably lower than market prices."

**Comment by ec-mcs** [comment]
From a technical point of view it could be a tick-box to indicate that it was a "Bargain Purchase". Do we know that this happens? The reason is that PP takes a long time so if something is available for a considerable low price it might not be available anymore.

**Comment by muricna** [comment]
The WG of 25/05/2018 agreed that Bargain purchase and value need to be separated.

To be treated in a future meeting

**Comment by JachymHercher** [comment]
(Responding to an off-GitHub question:)

In the eForms, bargain value is modeled as value for a notice where a procedure had a BT-136 (Direct Award Justification Code) equal to 

"Bargain purchase taking advantage of a particularly advantageous opportunity available for a very short time at a price considerably lower than market prices"
